Cargo.toml

```diff
@@ -19,8 +19,8 @@ grpcio-prost-codec = ["grpcio-compiler/prost-codec", "prost-codec"]
 proc-macro2 = { version = "1", optional = true }
 protobuf = { version = "2", optional = true }
 protobuf-codegen = { version = "2", optional = true }
-grpcio-compiler = { version = ">=0.5.0", default-features = false, optional = true }
-prost-build = { version = "0.6", optional = true }
+grpcio-compiler = { version = "0.8", default-features = false, optional = true }
```

Comment:
It was set to ">=0.5.0" on purpose in #43.
> Currently there is cycle dependencies between grpcio and protobuf-build.
To update grpcio-proto, protobuf-build has to be updated first to use
latest grpcio-compiler so that grpcio-proto can utilize latest syntax.
But grpcio-compiler is not released yet due to CI failure, which is
caused by outdated sources generated in grpcio-proto by protobuf-build.